server/src/main/java/org/opensearch/action/search/TransportSearchAction.java

```diff
@@ -906,7 +907,7 @@ private Index[] resolveLocalIndices(OriginalIndices localIndices, ClusterState c
     private void executeSearch(
         SearchTask task,
         SearchTimeProvider timeProvider,
-        SearchRequest searchRequest,
+        PipelinedRequest pipelinedRequest,
```

Comment:
can we rename class `PipelinedRequest` to `PipelinedSearchRequest` to be explicit. If we start supporting pipelines for other request types than it can become confusing

Reply:
@msfroh whats your thoughts on this?

Reply:
I have no objection. I also like the more explicit name. 

It's an internal class, so it's safe to change.